src/libs/CategoryUtils.ts

```diff
@@ -68,4 +68,14 @@ function getCategoryDefaultTaxRate(expenseRules: ExpenseRule[], categoryName: st
     return categoryDefaultTaxRate;
 }
 
-export {formatDefaultTaxRateText, formatRequireReceiptsOverText, getCategoryApproverRule, getCategoryExpenseRule, getCategoryDefaultTaxRate};
+function updateCategoryInMccGroups(mccGroups: never[] | Record<string, MccGroup>, oldCategoryName: string, newCategoryName: string): Record<string, MccGroup> {
+    const updatedGroups: Record<string, MccGroup> = {};
+
+    for (const [key, group] of Object.entries(mccGroups || {})) {
+        updatedGroups[key] = group.category === oldCategoryName ? {...group, category: newCategoryName, pendingAction: 'update'} : group;
```

Comment:
```suggestion
        updatedGroups[key] = group.category === oldCategoryName ? {...group, category: newCategoryName, pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE} : group;
```

Reply:
@eVoloshchak Thank you for the suggestion. I've updated it.